server/templates/obslist.html

```diff
@@ -21,14 +73,111 @@ <h3>Observations</h3>
             <td>{{ obs.aos }}</td>
             <td>{{ obs.tca }}</td>
             <td>{{ obs.los }}</td>
-            <td><a href="https://www.n2yo.com/satellite/?s={{ obs.sat_id }}">{{ obs.sat_name }}</a></td>
-            <td><a href="/station/{{ obs.station_id }}">{{ obs.station_id}}</a></td>
-            <td><a href="/obs/{{ obs.obs_id }}"> <img src="/data/thumbs/{{ obs.thumbnail }}"></a></td>
+            <td>
+                <a href="https://www.n2yo.com/satellite/?s={{ obs.sat_id }}">
+                    {{ obs.sat_name }}
+                </a>
+            </td>
+            <td>
+                <a href="/station/{{ obs.station_id }}">
+                    {{ obs.station_name }}
+                </a>
+            </td>
+            <td>
+                <a href="/obs/{{ obs.obs_id }}">
+                    <img src="/data/thumbs/{{ obs.thumbnail }}" />
+                </a>
+            </td>
         </tr>
+    {% else %}
+    <tr>
+        <td class="text-center" colspan="100%">No data for provided criteria</td>
+    </tr>
     {% endfor %}
 </table>
 </p>
 {% include 'pagination.html' %}
 
 {% endblock %}
 
+{% block scripts %}
+<script defer>
+    const form = document.getElementById("search-form");
+
+    function getFormControls()
+    {
+        return form.querySelectorAll("input, select");
+    }
+
+    function disableNonEmptyControls()
```

Comment:
Shouldn't this be disableEmptyControls? The code disables the fields that are empty.

Reply:
Right!

Reply:
Fixed.